populate_db.py

```diff
@@ -67,6 +68,21 @@ def create_services():
 def create_settings():
     get_or_create(Setting, app_name='Open Event')
 
+    if current_app.config['DEVELOPMENT']:
+        # get the stripe keys from the env file and save it in the settings.
+        env.read_envfile()
+        stripe_secret_key = env('STRIPE_SECRET_KEY', default=None)
```

Comment:
Yes, we don't run `heroku run cp ...` but heroku works differently. The environment variables are defined in something called `config vars`. You can find them in the settings tab on the project dashboard. Other environment variables such as app_secret is defined at that location only. So I think this will work.